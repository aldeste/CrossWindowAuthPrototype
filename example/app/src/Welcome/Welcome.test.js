import React from "react";
import Welcome from "./Welcome";
import ReactShallowRenderer from "react-test-renderer/shallow";
const renderer = new ReactShallowRenderer();

describe("Welcome react component", () => {
  const tree = renderer.render(<Welcome />);
  it("Is a react component", () => {
    expect(typeof Welcome).toBe("function");
    expect(tree.$$typeof.toString()).toBe("Symbol(react.element)");
  });

  it("Matches earlier configuration", () => {
    expect(tree).toMatchSnapshot();
  });
});
