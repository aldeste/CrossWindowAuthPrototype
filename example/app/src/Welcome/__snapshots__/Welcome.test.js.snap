// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome react component Matches earlier configuration 1`] = `
<styled.div>
  <styled.h1 />
  <styled.p>
    You're logged in as 
  </styled.p>
  <styled.button
    alternative={true}
  >
    Log out
  </styled.button>
</styled.div>
`;
